src/main/scala/org/constellation/snapshotstreaming/Configuration.scala

```diff
@@ -87,4 +87,17 @@ class Configuration(sharedConfigReader: SharedConfigReader) {
   val s3ApiRegion: Option[String] = Try(s3.getString("api.region")).toOption
   val s3ApiPathStyleEnabled: Option[Boolean] = Try(s3.getBoolean("api.pathStyleEnabled")).toOption
 
+
+}
+object Configuration {
+  import pureconfig.ConfigSource
+  import pureconfig.generic.auto._
+
+  case class DbConfig(host: String, port: Int, user: String, password: Option[String], database: String, maxSessions: Int)
+
+  lazy val dbConfig: DbConfig = ConfigSource
+    .default
+    .at("snapshotStreaming.db")
+    .loadOrThrow[DbConfig]
+
```

Comment:
can we use the pattern that we already have configured in the project? it would be great not to make two different ways of reading config, so either use the previous one or refactor the previous one to use pureconfig instead (but as a separate PR) 🤔

Reply:
This is a WIP and the main goal is to validate the approach and library. Once it reaches a mature state, I'll unify it with the rest of the project. I favor PureConfig because is less boilerplate.

Reply:
Ok. Me too, but EOD I'd opt for consistency so either this or that